addons/web/static/src/core/emoji_picker/emoji_picker.js

```diff
@@ -255,9 +332,12 @@ export class EmojiPicker extends Component {
             emojisToDisplay = emojisToDisplay.filter((emoji) => !recentEmojis.includes(emoji));
         }
         if (this.searchTerm.length > 1) {
-            return fuzzyLookup(this.searchTerm, emojisToDisplay, (emoji) =>
-                [emoji.name, ...emoji.keywords, ...emoji.emoticons, ...emoji.shortcodes].join(" ")
-            );
+            return fuzzyLookup(this.searchTerm, emojisToDisplay, (emoji) => [
+                emoji.name,
+                ...emoji.keywords,
+                ...emoji.emoticons,
+                ...emoji.shortcodes,
+            ]);
```

Comment:
Is the remove of `.join(" ")` intended?

Reply:
Yes, that's what improves the search in this PR. this makes the search reliable as it checks matches on each item rather than a weird concatenation of all items.

I think at the time of 16.3 refactoring the `fuzzyLookup` only allowed a string, but now it seems it allows array of strings which is definitely better.